pykechain/models/part.py

```diff
@@ -492,3 +492,27 @@ def as_dict(self):
             properties_dict[prop.name] = prop.value
         return properties_dict
 
+    def order_properties(self, list_property_names=None):
+        """
+        Order the properties of a part model using a list of property names.
+
+        Example
+        -------
+        >>> front_fork = client.scope('Bike Project').model('Front Fork')
+        >>> front_fork.order_properties(['Material', 'Height (mm)', 'Color'])
+
+        """
+        assert self.category == Category.MODEL
+
+        if isinstance(list_property_names, list):
```

Comment:
only names, or also property objects from pykechain.

Throughout the code we use mostly both. I agree that names should be 'unique', but there is no guarentee that these are. In that sense the ordering should also be done by a list of property (model) id's